assignment5.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs');
+
+var fileReader = function(fileName) {
+  return new Promise( (resolve, reject) => {
+    fs.readFile(fileName, 'utf8', function(error, data) {
+      console.log(data); 
+      console.log("============");
+
+      if(error)
+        return reject(fileName + 'file not Done');
+      resolve(fileName + 'fileDone');
```

Comment:
Are you using the string sent by `resolve`. If not, give a valid reason of having this string here.